index.html

```diff
@@ -203,7 +206,84 @@ <h3>Skills</h3>
         </div>
       </section>
 
-     
+      <section class="contact">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+
+        <form
+          action="https://formspree.io/f/xlezyjjy"
+          method="post"
+          class="form-action"
+        >
+          <input
+            type="text"
+            name="name"
+            placeholder="Full name"
+            maxlength="30"
+            required
+          />
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="Email address"
+            required
+          />
+          <textarea
+            name="message"
+            placeholder="Enter text here"
+            id="message"
+            cols="30"
+            rows="10"
+            maxlength="500"
+            required
+          ></textarea>
```

Comment:
Thanks Bondok6. I have impemented these features already. 
<img width="897" alt="image" src="https://user-images.githubusercontent.com/86082070/177496072-2f883258-4574-40f5-8447-2d401bbed9a3.png">
Also, the `textarea` border is supposed to be green onFocus, not permanently, and I have implemented that

Reply:
Hey @kendoriddy,

It's ok if you added a green border onFocus, but It doesn't appear because you have to remove the outline from the textarea and the inputs tags using `outline: none`,

```
input::focus,
textarea::focus {
  border: 1px solid #60c095 !important;
  border-radius: 4px;
  color: #091e42;
  outline: none; 
}
```